perf.env

```diff
@@ -114,10 +120,33 @@ CONF_CL_HANDLER_ADMIN=central-handlers.js
 ## QS - Quoting Service Config
 QS_CACHE__ENUM_DATA_EXPIRES_IN_MS=4170000
 QS_CACHE__PARTICIPANT_DATA_EXPIRES_IN_MS=60000
-QS_SWITCH_ENDPOINT="http://callback-handler-svc-cl-sim:3001/admin"
+# QS_SWITCH_ENDPOINT="http://sim-fspiop:3001/admin"
 # Use this for E2E testing
 # QS_SWITCH_ENDPOINT="http://central-ledger:3001"
 QS_SIMPLE_ROUTING_MODE=false
 
 # Use this for E2E testing
 # ALS_SWITCH_ENDPOINT="http://central-ledger:3001"
+
```

Comment:
Suggestion. Put these in there own sdk.env file. 
They don't have prefixes and they can cause confusion.
Or leave as is, you did give them header comments.